store/rootmulti/simple_mapt_test.go

```diff
@@ -0,0 +1,49 @@
+package rootmulti
```

Comment:
there's a typo on the file name: `simple_mapt_test.go` -> `simple_map_test.go`

Reply:
yup yup fixed in a different pr